### PR TITLE
integrated sendgrid api & added send email button

### LINKED ISSUE #11: Send email notification from admin dashboard
### Description
As an admin, I'd like to trigger an email notification for a specific user from the admin dashboard

### Acceptance Criteria
- Add a button "Send email" for each user 
- On click of the button send a simple email that says "This is a notification from Florish" to the user email address.
- We'll use Sendgrid for this. Ask Zee for credentials to our Sendgrid account.

### PR DESCRIPTION
**Pull request by beebzz**
Fixes #11

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,8 @@
     "setup": "node ./scripts/setup.js"
   },
   "dependencies": {
+    "@sendgrid/mail": "^7.4.2",
+    "fs": "^0.0.1-security",
```
Comment (zeeberry) [inline package.json:14]: do we still need this `fs` package?

**Review by zeeberry — COMMENTED** [review]
Very nice work! 

Added a super small comment. One other thing I'll add -- since this feature requires adding a new env variable, can you add instructions to the `README`?

File: package.json (hunk 1 above)
Comment (beebzz) [reply package.json:14]: Nope, good catch!

**Review by zeeberry — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**